client/app/scripts/charts/node.js

```diff
@@ -14,27 +14,27 @@ import { NODE_BASE_SIZE } from '../constants/styles';
 import NodeShapeStack from './node-shape-stack';
 import NodeNetworksOverlay from './node-networks-overlay';
 import {
-  NodeShapeCloud,
   NodeShapeCircle,
```

Comment:
oh, derp, I misread your comment. Thought you were talking about the cloud one, not circle. Yes, haha, a circle is uniform convergent to the limit as n->infinity of an n-sided regular polygon. It's also 1-sided if you consider a side to be a smooth line (ie. a curve with no 'corners').